example/lib/main.dart

```diff
@@ -4,23 +4,26 @@ import 'package:flutter_localizations/flutter_localizations.dart';
 import 'package:form_builder_validators/form_builder_validators.dart';
 import 'package:intl/intl.dart';
 
+import 'home_page.dart';
+
 void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
+    return const MaterialApp(
       title: 'Flutter FormBuilder Demo',
       debugShowCheckedModeBanner: false,
-      localizationsDelegates: const [
+      localizationsDelegates: [
         FormBuilderLocalizations.delegate,
         GlobalMaterialLocalizations.delegate,
         GlobalWidgetsLocalizations.delegate,
       ],
-      supportedLocales: FormBuilderLocalizations.delegate.supportedLocales,
-      home: const CompleteForm(),
+      // supportedLocales: FormBuilderLocalizations.delegate.supportedLocales,
+      home: HomePage(),
+      // home: const CompleteForm(),
```

Comment:
Please remove this comment line

Reply:
I think `HomePage` is more useful, and `CompleteForm` can not go other page, consider add some button？

Reply:
Yeah, for me is perfect made a Home page with buttons to both pages